lib/collectionspace/converter/publicart/organization.rb

```diff
@@ -3,54 +3,79 @@ module Converter
     module PublicArt
       class PublicArtOrganization < Organization
         ::PublicArtOrganization = CollectionSpace::Converter::PublicArt::PublicArtOrganization
+        def redefined_fields
+          @redefined.concat([
+            # not in publicart
+            'contactname',
+            'group',
+            'function',
+            # overridden by publicart
+            'foundingplace'
```

Comment:
@Paulmulonzia - this field is populated by authority in public art: local place or shared place, so I added two separate columns in the csv (foundingplacelocal and foundingplaceshared) and mapped them to the appropriate authority. Even though these columns should be in the CSV and no foundingplace column should be in the CSV, it's safer to list this here so it won't ever be pulled in/populated in error. (It also provides a nice overview of all core fields that have changed)